modules/travel_pay/app/services/travel_pay/client.rb

```diff
@@ -112,7 +112,7 @@ def connection(server_url:)
       Faraday.new(url: server_url) do |conn|
         conn.use :breakers
         conn.response :raise_error, error_prefix: service_name, include_request: true
-        conn.response :betamocks if use_fakes?
+        conn.response :betamocks if mock_enabled?
```

Comment:
why change this?

Reply:
`mock_enabled?` seemed to be a widely used pattern throughout the rest of `vets-api` for this scenario, so I switched it for consistency.